app/server.js

```diff
@@ -23,5 +23,5 @@ function simpleResponse(request, response) {
 };
 
 server.listen(serverPort, function() {
-	console.log('IT MIGHT BE WORKING?');
+	console.log('Unicorns are running');
```

Comment:
Love it ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ ❤️ 

Should we print out a small unicorn under the text???? 🐴